catalog/transformer/catalog-transformer-tika-input/src/test/java/ddf/catalog/transformer/input/tika/TikaInputTransformerTest.java

```diff
@@ -472,6 +474,9 @@ public void testMp3() throws Exception {
         containsString("<meta name=\"xmpDM:artist\" content=\"Test Artist\" />"));
     assertThat(metacard.getContentTypeName(), is("audio/mpeg"));
     assertThat(metacard.getAttribute(Core.DATATYPE).getValue(), is(SOUND));
+    assertThat(
+        metacard.getMetadata(), containsString("<meta name=\"xmpDM:duration\" content=\"2.4"));
+    assertTrue(metacard.getAttribute(Media.DURATION).getValue().toString().startsWith("2.4"));
```

Comment:
If this assertion fails, all the message will say is something like "expected true but was false", which isn't helpful in diagnosing the failure.

We should prefer `assertThat()`: `assertThat(metacard.getAttribute(Media.DURATION).getValue(), is(2.4))`

See https://codice.atlassian.net/wiki/spaces/DDF/pages/6291523/Unit+Testing#UnitTesting-Assertions